qiskit/transpiler/passes/calibration/builders.py

```diff
@@ -103,26 +103,32 @@ class RZXCalibrationBuilder(CalibrationBuilder):
     angle. Additional details can be found in https://arxiv.org/abs/2012.11660.
     """
 
-    def __init__(self, backend: basebackend):
+    def __init__(
+        self,
+        instruction_schedule_map: InstructionScheduleMap,
+        qubit_channel_mapping: List[List[str]],
+    ):
         """
         Initializes a RZXGate calibration builder.
 
         Args:
-            backend: Backend for which to construct the gates.
+            instruction_schedule_map: The :obj:`InstructionScheduleMap` object representing the
+                default pulse calibrations for the target backend
+            qubit_channel_mapping: The list mapping qubit indices to the list of
+                channel names that apply on that qubit.
 
         Raises:
             QiskitError: if open pulse is not supported by the backend.
         """
         super().__init__()
-        if not backend.configuration().open_pulse:
+        if instruction_schedule_map is None or qubit_channel_mapping is None:
             raise QiskitError(
                 "Calibrations can only be added to Pulse-enabled backends, "
                 "but {} is not enabled with Pulse.".format(backend.name())
             )
```

Comment:
name error on `backend` in the error

Reply:
Oops I pushed before committing my fixes from lint. It's updated now